backend/benefit/calculator/tests/factories.py

```diff
@@ -136,7 +136,7 @@ class PreviousBenefitFactory(factory.django.DjangoModelFactory):
         "pydecimal", left_digits=3, right_digits=2, min_value=0
     )
     total_amount = factory.LazyAttribute(
-        lambda o: duration_in_months(o.start_date, o.end_date)
+        lambda o: duration_in_months(o.start_date, o.end_date) * o.monthly_amount
```

Comment:
Has the meaning of the total_amount field changed? If it has then ok, adding the multiplication by monthly_amount seems logical. If it has not then was the multiplication left out previously by accident?

Reply:
yes, it was a bug

Reply:
Ok, this is clear